inference/similar.py

```diff
@@ -1,8 +1,3 @@
-"""
-Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-SPDX-License-Identifier: MIT-0
```

Comment:
What made you strip the license statement? Is this not derived from Amazon code?

Reply:
The derivation was: we started from the sample code, completely replaced 99% of that code with our own, then recombined chunks of our code into a new source file later on, and the license got copied in with our imports. We thought we had talked through it with you but maybe it fell off the agenda. We're open to leaving them in these files on your judgement